create_zpt/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/python:3.11
+
+# Install the function's dependencies using file requirements.txt
+# from your project folder.
+
+COPY ./requirements.txt  .
+RUN  pip3 install -r requirements.txt --target "${LAMBDA_TASK_ROOT}"
+
+# Copy function code
+COPY ./handler ${LAMBDA_TASK_ROOT}/handler
+
+# Set the CMD to your handler (could also be done as a parameter override outside of the Dockerfile)
+CMD [ "handelr.create_zpt_handler.handler" ] 
```

Kanske det vi ser i loggarna?
```suggestion
CMD [ "handler.create_zpt_handler.handler" ] 
```

---

Ja, det ser ju rimligt ut...